routerUtils.js

```diff
@@ -27,6 +27,7 @@ async function sendToDestination(destination, payload) {
   if (resp.success) {
     const { response } = resp;
     parsedResponse = {
+      headers: response.headers,
```

Comment:
why is this added ?

Reply:
for checking the response type. For eg: if the content-type is image/gif, we are setting the response to an empty string.

Reply:
Got it 👍🏼